client/packages/common/src/ui/components/domain/Dashboard/StatsPanel.tsx

```diff
@@ -23,7 +23,7 @@ export interface StatsPanelProps {
 const Statistic: FC<Stat> = ({ label, value, link }) => {
   const t = useTranslation();
   return (
-    <Grid container alignItems="center" style={{ height: 30 }}>
```

Comment:
Remove fixed height of each satistic so that they can wrap text and won't sit on top of each other

Reply:
The Margin top adds spacing so that for single line text it doesn't look too cramped